tests/integration-tests/tests/performance_tests/test_openfoam.py

```diff
@@ -0,0 +1,82 @@
+import json
+import logging
+
+import boto3
+import pytest
+from assertpy import assert_that
+from remote_command_executor import RemoteCommandExecutionError, RemoteCommandExecutor
+
+# timeout in seconds
+OPENFOAM_INSTALLATION_TIMEOUT = 1800
```

Comment:
30 minutes it's too much for the OpenFoam installation, because the installation script is nothing more than download archive + extract from S3. I suggest 5 min at most